fe/fe-core/src/main/java/org/apache/doris/job/extensions/mtmv/MTMVTask.java

```diff
@@ -97,7 +103,7 @@ public enum MTMVTaskTriggerMode {
 
     public enum MTMVTaskRefreshMode {
         COMPLETE,
-        PARTITION,
+        PARTIAL,
```

Comment:
why change to partial? how to distingush beteen partition refresh and binlog data refresh if we named partition refrash as partial?

Reply:
`COMPLETE ，PARTIAL ，NOT_REFRESH`is the same dimension,`PARTITION `and`binlog ` is another dimension